packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -77,7 +77,21 @@ class Question {
 	 */
 	url;
 
-	constructor({ title, question, description, viewFolder, fieldName, validators, url } = {}) {
+	/**
+	 * @type {string} optional html content
+	 */
+	html;
+
+	constructor({
+		title,
+		question,
+		description,
+		viewFolder,
+		fieldName,
+		validators,
+		url,
+		html = '/blank.html'
```

Comment:
What about instead of a default value in the nun jucks template we did something like? 
`{% if showBackToListLink %}
{% block optional_html %}
    {% include question.html ignore missing %}
{% endblock optional_html %}
{% endif %}`

Reply:
Apologies, yes exactly that. Thats the example value from the nun jucks docs that I forgot to replace sorry.